core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.component.html

```diff
@@ -173,7 +173,7 @@ <h3 tabindex="0">Content</h3>
             </oppia-response-header>
           </a>
           <div *ngIf="activeCustomizationArgContentIndex === index">
-            <div class="oppia-editor-card-section" *ngIf="caContent.content._html">
+            <div class="oppia-editor-card-section" *ngIf="caContent.content">
```

Comment:
Would this result in the content showing twice as the ng-if in line 190 is specific to unicode text? i.e. translated and a "Continue" below it.

I think the expectation is that only the translated text shows.

Reply:
@kevintab95  I have addressed all the comments.